WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -20,18 +21,20 @@ namespace WalletWasabi.WabiSabi.Client
 {
 	public class CoinJoinManager : BackgroundService
 	{
-		public CoinJoinManager(WalletManager walletManager, RoundStateUpdater roundStatusUpdater, HttpClientFactory backendHttpClientFactory, ServiceConfiguration serviceConfiguration)
+		public CoinJoinManager(WalletManager walletManager, SystemAwakeChecker? systemAwakeChecker, RoundStateUpdater roundStatusUpdater, HttpClientFactory backendHttpClientFactory, ServiceConfiguration serviceConfiguration)
```

Comment:
Would it be better to decouple `CoinJoinManager` & `SystemAwakeChecker` by having an event or a simple callback?

Reply:
ACK! Done